nullaway/src/main/java/com/uber/nullaway/handlers/BaseNoOpHandler.java

```diff
@@ -59,6 +62,19 @@ protected BaseNoOpHandler() {
     // We don't allow creating useless handlers, subclass to add real behavior.
   }
 
+  protected static void reportMatch(
+      NullAway analysis, VisitorState state, Tree errorLocTree, String message) {
+    assert analysis != null && state != null;
```

Comment:
Make this `Preconditions.checkNotNull` instead.  We don't generally test NullAway with assertions enabled